src/main.rs

```diff
@@ -15,11 +15,21 @@ fn main() -> Result<(), Box<dyn Error>> {
     }
 
     let warnings = dotenv_linter::run(args, &current_dir)?;
-    if warnings.is_empty() {
+
+    let mut all_fixed = true;
```

Comment:
It seems to me, for Rust, it will be more idiomatic to use functional programming approaches instead of a mutable variable (`filter()`, `map()`,  `filter_map()`, `any()` and others) :eyes:

Reply:
I am agree about the functional style.
But first we need to finally decide how we will display warnings (format).

Reply:
I changed the output format and there is no `all_fixed` now.